diagnostic.md

```diff
@@ -9,6 +9,13 @@ Assume all code executes in strict mode.
 
 Please list the JavaScript primitives and give one example of each.
 
+1. Numbers (always a number) or Infinity,  or 0
+2. strings ‘Ryan Roderick’  (always quoted, single or double) (can also be ‘ ‘ empty)
+3. Boolean (statement types TRUE & FALSE equations, <, >, =, etc.
+4. Null (Only has one value)
+5. Undefined (Only has one value)
```

Comment:
Yes, null and undefined only have 1 value, but what are the values? Please read the question carefully!